lib/feature/add_character/add_character_page.dart

```diff
@@ -0,0 +1,134 @@
+import 'package:character_viewer/feature/add_character/cubit/add_character_cubit.dart';
+import 'package:character_viewer/feature/add_character/cubit/add_character_state.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class AddCharacterPage extends StatelessWidget {
+  const AddCharacterPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Add Character'),
+      ),
+      body: BlocProvider(
+        create: (context) => CharacterCubit(),
+        child: AddCharacterForm(),
+      ),
+    );
+  }
+}
+
+class AddCharacterForm extends StatelessWidget {
+  final TextEditingController _nameController = TextEditingController();
```

Comment:
when we work with controllers and stuff we should use stful widgets

Reply:
it'll work in both cases but with stless we can have some memory leaks in the future